src/Aspire.Hosting/build/Aspire.Hosting.targets

```diff
@@ -4,6 +4,7 @@
     <!-- Similar to ASP.NET Core and Worker apps, set the default CWD of orchestrator projects to be the project directory. -->
     <RunWorkingDirectory Condition=" '$(RunWorkingDirectory)' == '' and '$(EnableDefaultRunWorkingDirectory)' != 'false' ">$(MSBuildProjectDirectory)</RunWorkingDirectory>
     <_AspireIntermediatePath>$(IntermediateOutputPath)Aspire\</_AspireIntermediatePath>
+    <OutputType Condition=" '$(IsAspireHost)' == 'true' ">Exe</OutputType>
```

Comment:
This is too late to be setting this value. There is MSBuild logic that get imported before Aspire.Hosting.targets that checks for OutputType. For example, `CheckEolTargetFramework` gets evaluated before this. https://github.com/dotnet/sdk/blob/e4001e7c1ffe9ee5f0f919f5c228811934ce1fa7/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.EolTargetFrameworks.targets#L15.

Let's leave the OutputType in the .csproj as it is now, so we don't break anything.